### ISSUE TITLE
Input images missing from the dataset

### ISSUE THREAD

**Issue by anthonymakela** [zlzeng/DeepFloorplan#8]
Hi!

It seems that the input images such as "'../dataset/jp/train/100_input.jpg'" are missing from the "jp" dataset. Is there any way to acquire them? @zlzeng

**Comment by mountgrass** [comment]
Same question here. 

For those who are able to run the code, could you give us some guidance on how to setup the pipeline? Could you share your python and tensorflow version that can run the code? 
Thank you!

**Comment by TerenceWangh** [comment]
> Hi!
> 
> It seems that the input images such as "'../dataset/jp/train/100_input.jpg'" are missing from the "jp" dataset. Is there any way to acquire them? @zlzeng

You should apply the dataset of japan first.

> Same question here.
> 
> For those who are able to run the code, could you give us some guidance on how to setup the pipeline? Could you share your python and tensorflow version that can run the code?
> Thank you!

I run the code with python 2.7 and tensorflow 1.10.0.
and use the loss of softmax_cross_entropy_with_logits_v2. And I can't get result using the balanced_entropy.

**Comment by zhuao1997** [comment]
@TerenceWangh  Hello, do you know how to get the complete R2V data set?

**Comment by JanineCHEN** [comment]
> @TerenceWangh Hello, do you know how to get the complete R2V data set?

Same question here, all the #_input.jpg images seems missing from the provided dataset. @zlzeng

**Comment by TerenceWangh** [comment]
@zhuao1997 @JanineCHEN you may refer the website https://lifull.com/ and send email to apply the dataset. And send email to the authors to get the txt of train list.

**Comment by Flyingdog-Huang** [comment]
> @zhuao1997 @JanineCHEN you may refer the website https://lifull.com/ and send email to apply the dataset. And send email to the authors to get the txt of train list.

hi, if I want to get the R2V data set , Who should I send the application email to? THX
